inc/saitypes.h

```diff
@@ -157,51 +157,51 @@ typedef enum _sai_common_api_t {
-    SAI_OBJECT_TYPE_SWITCH                  = 32,
-    SAI_OBJECT_TYPE_HOSTIF_TRAP             = 33,
-    SAI_OBJECT_TYPE_HOSTIF_TRAP_USER_DEF    = 34,
-    SAI_OBJECT_TYPE_NEIGHBOR_ENTRY          = 35,
-    SAI_OBJECT_TYPE_ROUTE_ENTRY             = 36,
-    SAI_OBJECT_TYPE_VLAN                    = 37,
-    SAI_OBJECT_TYPE_VLAN_MEMBER             = 38,
-    SAI_OBJECT_TYPE_PACKET                  = 39,
-    SAI_OBJECT_TYPE_TUNNEL_MAP              = 40,
-    SAI_OBJECT_TYPE_TUNNEL                  = 41,
-    SAI_OBJECT_TYPE_TUNNEL_TERM_TABLE_ENTRY = 42,
-    SAI_OBJECT_TYPE_INGRESS_PRIORITY_GROUP  = 43,
-    SAI_OBJECT_TYPE_MAX                     = 44
+    SAI_OBJECT_TYPE_NULL                     =  0, /**< invalid object type */
+    SAI_OBJECT_TYPE_PORT                     =  1,
+    SAI_OBJECT_TYPE_LAG                      =  2,
+    SAI_OBJECT_TYPE_VIRTUAL_ROUTER           =  3,
+    SAI_OBJECT_TYPE_NEXT_HOP                 =  4,
+    SAI_OBJECT_TYPE_NEXT_HOP_GROUP           =  5,
+    SAI_OBJECT_TYPE_ROUTER_INTERFACE         =  6,
+    SAI_OBJECT_TYPE_ACL_TABLE                =  7,
+    SAI_OBJECT_TYPE_ACL_ENTRY                =  8,
+    SAI_OBJECT_TYPE_ACL_COUNTER              =  9,
+    SAI_OBJECT_TYPE_ACL_RANGE                = 10,
+    SAI_OBJECT_TYPE_HOSTIF                   = 11,
+    SAI_OBJECT_TYPE_MIRROR_SESSION           = 12,
+    SAI_OBJECT_TYPE_SAMPLEPACKET             = 13,
+    SAI_OBJECT_TYPE_STP                      = 14,
+    SAI_OBJECT_TYPE_HOSTIF_TRAP_GROUP        = 15,
+    SAI_OBJECT_TYPE_ACL_TABLE_GROUP          = 16,
+    SAI_OBJECT_TYPE_POLICER                  = 17,
+    SAI_OBJECT_TYPE_WRED                     = 18,
+    SAI_OBJECT_TYPE_QOS_MAP                  = 19,
+    SAI_OBJECT_TYPE_QUEUE                    = 20,
+    SAI_OBJECT_TYPE_SCHEDULER                = 21,
+    SAI_OBJECT_TYPE_SCHEDULER_GROUP          = 22,
+    SAI_OBJECT_TYPE_BUFFER_POOL              = 23,
+    SAI_OBJECT_TYPE_BUFFER_PROFILE           = 24,
+    SAI_OBJECT_TYPE_INGRESS_PRIORITY_GROUP   = 25,
+    SAI_OBJECT_TYPE_LAG_MEMBER               = 26,
+    SAI_OBJECT_TYPE_HASH                     = 27,
+    SAI_OBJECT_TYPE_UDF                      = 28,
+    SAI_OBJECT_TYPE_UDF_MATCH                = 29,
+    SAI_OBJECT_TYPE_UDF_GROUP                = 30,
+    SAI_OBJECT_TYPE_FDB_ENTRY                = 31,
+    SAI_OBJECT_TYPE_SWITCH                   = 32,
+    SAI_OBJECT_TYPE_HOSTIF_TRAP              = 33,
+    SAI_OBJECT_TYPE_HOSTIF_USER_DEFINED_TRAP = 34,
+    SAI_OBJECT_TYPE_NEIGHBOR_ENTRY           = 35,
+    SAI_OBJECT_TYPE_ROUTE_ENTRY              = 36,
+    SAI_OBJECT_TYPE_VLAN                     = 37,
+    SAI_OBJECT_TYPE_VLAN_MEMBER              = 38,
+    SAI_OBJECT_TYPE_HOSTIF_PACKET            = 39,
+    SAI_OBJECT_TYPE_TUNNEL_MAP               = 40,
+    SAI_OBJECT_TYPE_TUNNEL                   = 41,
+    SAI_OBJECT_TYPE_TUNNEL_TERM_TABLE_ENTRY  = 42,
+    SAI_OBJECT_TYPE_FDB_FLUSH                = 43,
```

Comment:
it's not really an object, but will be used to serialize fdb_flush in the future, same usecase as in hostif packet for recv/send